app/components/Common/PremiumPaywall.tsx

```
@@ -12,6 +12,113 @@ interface PremiumPaywallProps {
+// Component for when session is refreshing
+const DiscordSessionExpired: React.FC = () => (
+  <>
+    <DiscordIcon className="w-16 h-16 text-[#5865F2] mb-4" />
+    <h2 className="text-2xl font-bold mb-2 text-center">
+      Refreshing Session...
+    </h2>
+    <p className="mb-6 text-center text-gray-600 dark:text-gray-300">
+      Your Discord session has expired. Automatically refreshing your roles...
+    </p>
+    <div className="flex items-center justify-center">
+      <div className="animate-spin rounded-full h-8 w-8 border-b-2 border-[#5865F2]"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)